src/styles/blocks/stars.css

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  align-items: flex-end;
+  margin-top: 16px;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  align-items: center;
+  background-position: center;
+}
+
+.stars__star + .stars__star {
+  margin-left: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.card__review-count {
```

Comment:
it should be in the card component

Reply:
Not fixed, please pay more attention to comments and try to fix all of them before making re-request.